scalableminds/webknossos
# Enable hybrid tracings by default

This PR also adds the current "Tracing Type" to the info tab (volume/skeleton/hybrid). For volume/skeleton explorational! tracings, a button is then displayed to convert the tracing to a hybrid tracing.

### Missing:
- [x] Use existing segmentation layer when making skeleton -> hybrid tracing. /cc @youri-k 
- [x] Do we need to remove the hybridTracings flag from the templates in the kube repo?! - _No can be done later_

### URL of deployed dev instance (used for testing):
- https://enablehybridtracings.webknossos.xyz

### Steps to test:
- Open an explorational volume/skeleton tracing. Click the convert to hybrid button. The page should reload and the tracing should be a hybrid tracing. Brushing/Setting nodes etc. should work as expected.
- Open a dataset in view mode. The button should not be there.
- Open a task. The button should not be there.

### Issues:
- part of #3288

------
